cypress/integration/retailer/1-login-page.js

```diff
@@ -0,0 +1,14 @@
+/// <reference types="cypress" />
+
+describe('Testing login page', () => {
+  beforeEach(() => {
+    cy.kcLogout();
+    cy.visit('/retailer')
+  })
+
+  it("Displays the login page when logged out", () => {
+    cy.contains("E-Substances Reporting Application");
+    cy.contains("Continue with Basic BCeID");
+    cy.contains("Continue with Business BCeID");
```

Comment:
no changes needed - just a helpful hint: if you can deterministically determine one piece of unique content on a page to guarantee you're in the right place, you only need to do that one check